contrib/k8s.io/client-go/kubernetes/kubernetes.go

```diff
@@ -21,11 +22,20 @@ func WrapRoundTripper(rt http.RoundTripper) http.RoundTripper {
 	return httptrace.WrapRoundTripper(rt,
 		httptrace.WithBefore(func(req *http.Request, span ddtrace.Span) {
 			span.SetTag(ext.ServiceName, "kubernetes")
-			span.SetTag(ext.ResourceName, requestToResource(req.Method, req.URL.Path))
+			span.SetTag(ext.ResourceName, RequestToResource(req.Method, req.URL.Path))
+			traceID := span.Context().TraceID()
+			if traceID == 0 {
+			    // tracer is not running
+				return
```

Comment:
What's wrong with the alignment here?

Reply:
I accepted the github suggestion 😆  going to fix it in the old way

Reply:
Ok then I'll take the fall for that 😅